Source/Testably.Abstractions.Testing/FileSystem/FileSystemWatcherMock.cs

```diff
@@ -538,28 +548,12 @@ private string TransformPathAndName(
 		}
 
 		name = transformedName;
-		if (!_fileSystem.Path.IsPathRooted(Path))
-		{
-			string rootedWatchedPath = _fileSystem.Path.GetFullPath(Path);
-			if (path?.StartsWith(rootedWatchedPath, _fileSystem.Execute.StringComparisonMode) ==
-			    true)
-			{
-				path = _fileSystem.Path.Combine(Path, path.Substring(rootedWatchedPath.Length));
-			}
-		}
-
 		return path ?? "";
 	}
 
 	private void TriggerRenameNotification(ChangeDescription item)
 	{
-		if (!item.Path.StartsWith(Path, _fileSystem.Execute.StringComparisonMode) &&
-		    item.OldPath != null)
-		{
-			Deleted?.Invoke(this, ToFileSystemEventArgs(
-				WatcherChangeTypes.Deleted, item.OldPath, item.OldName));
-		}
-		else if (_fileSystem.Execute.IsWindows)
```

Comment:
Also added in #866 but again I didn't find it necessary. It seems that raising Created and Deleted notifications on renames is a Windows-only behaviour?